ui/src/components/records/RecordFilePreview.svelte

```diff
@@ -8,35 +8,35 @@
 
     let previewPopup;
     let thumbUrl = "";
-    let originalUrl = "";
-
-    $: hasPreview = CommonHelper.hasImageExtension(filename);
-
-    $: if (hasPreview) {
-        originalUrl = ApiClient.getFileUrl(record, `${filename}`);
-    }
+    let originalUrl = ApiClient.getFileUrl(record, `${filename}`);
 
+    $: type = CommonHelper.getFileType(filename);
+    $: hasPreview = /^(image|pdf)$/.test(type);
     $: thumbUrl = originalUrl ? originalUrl + "?thumb=100x100" : "";
 
     function onError() {
         thumbUrl = "";
     }
 </script>
 
-{#if hasPreview}
-    <img
-        src={thumbUrl}
-        alt={filename}
-        title="Preview {filename}"
-        class:link-fade={hasPreview}
-        on:click={(e) => {
-            e.stopPropagation();
-            previewPopup?.show(originalUrl);
-        }}
-        on:error={onError}
-    />
-{:else}
-    <i class="ri-file-line" />
-{/if}
-
-<PreviewPopup bind:this={previewPopup} />
+<a
+    class="thumb thumb-sm link-fade"
```

Comment:
The classes will interfere in other places like the preview in the Record create/update panel where we use a different thumb size so we need to make it configurable if we want to keep it this way.

One way could be to define a `export let size = ''; // sm/lg` prop and change the tag to:

```html
<a
    class="thumb thumb-{size} link-fade"
```